runtime/doc/repeat.txt

```diff
@@ -139,17 +139,78 @@ q			Stops recording.  (Implementation note: The 'q' that
 			result of evaluating the expression is executed as an
 			Ex command.
 			Mappings are not recognized in these commands.
-			{Vi: only in some versions} Future: Will execute the
-			register for each line in the address range.
+
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+			LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME LOOK AT ME
+
+			TODO(Pyrohh): what does the below line mean? I don't
+			know what is meant by "future", as I'm not sure if
+			what is mentioned below is already implemented.
```

Comment:
> 1) it doesn't provide useful information to the user, 
> 2) who knows when and if those features will be really implemented.

I think this is most important. If it's not dealt with not it will probably be in there for years.